### ISSUE TITLE
Confusion between https://github.com/krasileva/plant_rgenes

### ISSUE THREAD

**Issue by peterjc** [krasileva-group/plant_rgenes#4]
The paper https://doi.org/10.1186/s12915-016-0228-7 cites https://github.com/krasileva/plant_rgenes but that repository seems inactive with its issue tracker disabled.

Should https://github.com/krasileva-group/plant_rgenes be used instead? If so could you ask GitHub to setup a redirection?

**Comment by krasileva** [comment]
Yes, we migrated the scripts to the group directory as more people started to contribute and take ownership.
I was under impression there was a re-direct already, we will look into this.

Best wishes,

Ksenia

----------------------------------------------

Ksenia Krasileva
Group Leader

[cid:image001.png@01D35484.AAD5A220]

Norwich Research Park
Norwich
Norfolk
NR4 7UZ
+44 (0) 1603 450 2946
Ksenia.Krasileva@earlham.ac.uk<mailto:Ksenia.Krasileva@earlham.ac.uk>

www.earlham.ac.uk/krasileva-group<http://www.earlham.ac.uk/krasileva-group>
www.tsl.ac.uk/groups/krasileva-group<http://www.tsl.ac.uk/groups/krasileva-group>
@kseniakrasileva

www.earlham.ac.uk<http://www.earlham.ac.uk/>
www.tsl.ac.uk<http://www.tsl.ac.uk/>

[cid:image002.png@01D35484.AAD5A220]<https://twitter.com/EarlhamInst>[cid:image003.png@01D35484.AAD5A220]<https://www.facebook.com/EarlhamInst/>
